test/__mocks__/cozy-keys-lib/index.js

```diff
@@ -0,0 +1,6 @@
+const keysLib = jest.requireActual('cozy-keys-lib')
+
+module.exports = {
+  withVaultClient: x => x,
```

Comment:
Is it a new method?

Reply:
Nope but before cozy-harvest-lib did not import cozy-keys-lib so we did not have the problem.